opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -188,6 +188,15 @@ message Metric {
     ExponentialHistogram exponential_histogram = 10;
     Summary summary = 11;
   }
+
+  // Additional attributes that describe the metric. [Optional].
+  // Attributes are non-identifying.
+  // Consumers SHOULD not need to be aware of these attributes.
+  // These attributes MAY be used to encode information allowing
+  // for lossless roundtrip translation to / from another data model.
+  // Attribute keys MUST be unique (it is not allowed to have more than one
+  // attribute with the same key).
+  repeated opentelemetry.proto.common.v1.KeyValue attributes = 12;
```

Comment:
Do we expect that these will use attribute names from the same namespace as all other attributes? For product- or protocol-specific attributes we can just follow the regular attribute naming recommendation to use reverse FQDN, e.g. `io.prometheus.metric_type=unknown`.
This way we can safely use these attributes for any other orthogonal purposes knowing there won't be conflicts.

Reply:
>We might want to loosen the language to include "OSS Projects" if prometheus should be using a reverse fqdn. Right now, it seems to imply only individual companies need a reverse fqdn.

Yes, definitely. Any organization, whether a company or an OSS project.

>Do you think that should be documented in the proto?

I don't think we document this for any other attributes, so I am inclined to say no, unless we also document similarly other attributes fields. On the other hand all other attributes originate in the API/SDK, so it is clear that spec's attribute conventions apply to them, so perhaps for this one we should explicitly document it needs to follow the same rules as everything else. I am not sure what's the best way.

Reply:
I'm ok excluding it for now, unless others see a need.  It seems clear enough to me that attribute naming applies to all otel attributes.